# serial_bridge.param.yaml

serial_bridge:
  ros__parameters:

    # Serial port settings
    port_path: "/dev/ttyUSB0"
    baud_rate: 115200

    # Henes T870 parameters
    max_speed_mps      :  5.50 # m/s
    max_steering_deg   : 20.00 # deg
    steering_offset_deg:  0.00 # deg